runtime/runtime_test.go

```diff
@@ -304,7 +304,8 @@ func TestCheckOPAUpdateLoopWithNewUpdate(t *testing.T) {
 }
 
 func TestCheckAuthIneffective(t *testing.T) {
-	ctx, cancel := context.WithCancel(context.Background())
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Millisecond)
```

Comment:
I guess this timeout is arbitrary? We're just starting the server and then telling it to shutdown after 2 milliseconds. It's not like we need some time for the log to show up or anything...right?

Reply:
Just moved from below.